storage/sql/template.go

```diff
@@ -68,7 +80,11 @@ func NewDataSourceBuilderFactoryFromTemplate(template SQLSourceTemplate) func([]
 
 				// Create a query with placeholders to prepare a statement from a physical formula
 				query := FormulaToSQL(filter, placeholders)
-				query = fmt.Sprintf("SELECT * FROM %s %s WHERE %s", tableName, alias, query)
+
+				// Adding Offset placeholder
+				offsetPlaceholder := placeholders.AddPlaceholder(physical.NewVariable(octosql.NewVariableName("offset")))
+
```

Comment:
I would randomize this somehow in case there is a variable called "offset" that appears in the Formula.
Or call this something like $offset_placeholder$